services/director-v2/src/simcore_service_director_v2/core/events.py

```diff
@@ -11,7 +11,7 @@
 | | | |_ _ __ ___  ___| |_ ___  _ __
 | | | | | '__/ _ \/ __| __/ _ \| '__|
 | |/ /| | | |  __/ (__| || (_) | |
-|___/ |_|_|  \___|\___|\__\___/|_|   {0}
+|___/ |_|_|  \___|\___|\__\___/|_|   {}
```

Comment:
are you sure about this?

Reply:
yes. this is automatically done by pyupgrade